compiler/lib/testdata/src/enum_test.pconf

```diff
@@ -1,10 +1,16 @@
-load("test.proto", "MessageWithEnum")
+load("test.proto", "MessageWithEnum", "MessageWithExternalEnum", "TopLevelEnum")
 
 def main():
-    m=MessageWithEnum(e=MessageWithEnum.TestEnum.ONE)
+    m = MessageWithEnum(e=MessageWithEnum.TestEnum.ONE)
     if m.e == MessageWithEnum.TestEnum.TWO:
         fail("%s and %s should not match" % (m.e, MessageWithEnum.TestEnum.TWO))
     if m.e != MessageWithEnum.TestEnum.ONE:
         fail("%s and %s should match" % (m.e, MessageWithEnum.TestEnum.ONE))
 
+    m2 = MessageWithExternalEnum(e=TopLevelEnum.ONE)    
```

Comment:
Can you create another test file in which `main()` returns this value? This will make sure the message is being correctly serialized.

Reply:
Makes sense.  Done.